DataFormats/Math/interface/approx_log.h

```diff
@@ -36,10 +36,10 @@ please also keep these comments attached to the code of approx_logf.
 #define APPROX_MATH_N
 namespace approx_math {
   union binary32 {
-    binary32() : ui32(0) {};
```

Comment:
remind me please why are these ```#ifndef APPROX_MATH_N ... ``` not in a separate file instead of the present copy-pasted re-definition

Reply:
Ok, Let's take this opportunity of recompilation of the universe to clean these includes...
will do today